CONTRIBUTING.md

```diff
@@ -1,195 +1,119 @@
-
-## Contributing code
-
-### Accepting Contributor License Agreement (CLA)
-
-Before you submit your first contribution please accept our Contributor License Agreement (CLA) here. When you send your first Pull Request (PR), GitHub will ask you to accept the CLA.
-
-There is no need to do this before you send your first PR.
-
-Subsequent PRs will not require CLA acceptance.
-
-If for some (unlikely) reason at any time CLA changes, you will get presented with the new CLA text on your first PR after the change.
-
-### Obtaining the Source Code and Building the Project
-
-#### Build Prerequisites
-
-**Prerequisite 1** - Install a Java Development Kit (JDK) version 8
-
-Building Ballerina requires a Java SE Development Kit (JDK) version 8 to be installed. You can download one from one of the following locations:
-
-* [http://www.oracle.com/technetwork/java/javase/downloads/index.html](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
-* [http://openjdk.java.net/install/index.html](http://openjdk.java.net/install/index.html)
-
-> NOTE: Set an environment variable `JAVA_HOME` to the path name of the directory into which you installed the JDK release.
-
-**Prerequisite 2** - Node (v8.9.x or latest LTS release) + npm (v5.6.0 or later)
-
-**Prerequisite 3** - Docker 
-
-#### Building from the source
-
-Clone this repository using the following command.
-
-```
-git clone --recursive https://github.com/ballerina-platform/ballerina-lang.git
-```
-
-If you have forked the repository to your github account then use the following command replacing <YOUR-GITHUB-USERNAME> with your git username.
-
-````
-git clone --recursive https://github.com/<YOUR-GITHUB-USERNAME>/ballerina-lang.git
-````
-
-If you download the sources, you need to update the git submodules using the following command.
-
-```
-git submodule update --init 
-```
-
-Run the gradle command `./gradlew clean build` from the Ballerina root directory:
-
-Extract the Ballerina distribution created at `distribution/zip/ballerina/target/ballerina-<version>-SNAPSHOT.zip`. The `zip/ballerina` directory contains the runtime only. `zip/ballerina-tools/` contains the runtime and tools (e.g., Ballerina Composer).
-
-Note: It is possible to face an IOException error stating "Too many open files". This is due to the default number of possible open files being set to a lower number on your operating system than required for Ballerina to be compiled. You may have to increase the number of open files/file descriptors (FD) on your operating system. This is OS dependent and you can search the internet on how to do this for your operating system (Windows/OSX/Linux). You could update the maximum number of open files to 1000000 (or higher).
-
-### Setting up your development environment
-
-#### Setup IntelliJ IDEA
+- [Get started](#get-started)
+- [Set up the Prerequisites](#set-up-the-prerequisites)
+- [Obtain the source code](#obtain-the-source-code)
+- [Build the project](#build-the-project)
```

Comment:
Link is not working.

Reply:
Fixed as : - [Build the source code](#build-the-source-code)